source-data/country-codes.tsv

```diff
@@ -0,0 +1,248 @@
+# ISO 3166-1 alpha-3 codes
```

Comment:
The names here are non-standard and it seems like there are some missing codes too. Is there a reason to mangle the names?

For comparison, this is a standard list as pulled from Wikipedia (with two small name tweaks for disambiguation/geopolitical reasons): https://gist.github.com/tsibley/c5916b7d738e9826f8cdc9669a9d2543

Reply:
Mangled some to fit what I've seen in the GISAID data. If we don't care about matching the strain names between the two sources, then we can use the standard.

Reply:
I think we landed on "don't care about matching", and we can always mangle the names as part of any future matching process itself.

I'd definitely prefer to start from a standard point and mangle, rather than try to unmangle later.